### DISCUSSION TITLE
Layout shift when using a component that removes scrolling. `Dialog`

### DISCUSSION THREAD

**Discussion by AbdallahAbis** [radix-ui/primitives#1340] [Help]
I'm trying to use the Dialog primitive, but it's a causing layout shift cause it's applying both padding and margin on the right or could be for some other reason.

![Screen Shot 2022-04-22 at 20 12 12](https://user-images.githubusercontent.com/59185227/164790605-35e253d2-2921-492d-aecd-d2d13c6566d9.png)

I have had a look at the code and also checked other discussions and seems you're using `react-remove-scroll` to remove the scrollbar, and you've also pointed to using the `RemoveScroll.classNames.zeroRight` class name, but I don't know where exactly I should add that className, is it on the Dialog Root, or on the body, or the wrapper of the Dialog, or what?

**Comment by benoitgrelard** [comment]
Hey @AbdallahAbis that should be handled normally for you.
Do you have a sandbox with a reproducible example?

Thanks!

**Comment by predaytor** [comment]
Same here. I only got it on Windows.

**Comment by fazlehadii** [comment]
come on!!! some one answer this please i need it! exact same issue!
